test/api/v3/integration/groups/GET-groups.test.js

```diff
@@ -0,0 +1,51 @@
+import {
+  generateUser,
+} from '../../../../helpers/api-integration.helper';
+
+describe('GET /groups', () => {
+  let user;
+
+  beforeEach(async () => {
+    user = await generateUser({balance: 4});
+  });
+
+  it('returns a list of groups', async () => {
+    let groups = [
+      {
+        name: 'Test Party',
+        type: 'party',
+      },
+      {
+        name: 'Test Public Guild',
+        type: 'guild',
+      },
+      {
+        name: 'Test Private Guild',
+        type: 'guild',
+        privacy: 'private',
+      },
+    ];
+
+    let group1 = await user.post('/groups', {
+      name: groups[0].name,
+      type: groups[0].type,
+    });
+
+    let group2 = await user.post('/groups', {
+      name: groups[1].name,
+      type: groups[1].type,
+    });
+
+    let group3 = await user.post('/groups', {
+      name: groups[2].name,
+      type: groups[2].type,
+    });
+
+    expect(group1).toExist;
```

Comment:
These three expectations shouldn't be necessary. If there's an error, it'll get thrown and mocha will tell you about it. If it's not actually making  the groups, that's a concern of the post route, not this test.